test/container/php/test_config.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+
+CONFIG_DIR = '/usr/local/etc/php'
+
+@pytest.mark.php
+def test_php_ini_dir_environment_variable_is_set(host):
+    assert CONFIG_DIR in host.run('echo $PHP_INI_DIR').stdout
```

Comment:
you could be more strict than `in`, because it might have been moved to a sub directory or so
for instance: (also the "actual is expected" order)
```python
assert host.run('echo -n $PHP_INI_DIR').stdout is CONFIG_DIR
```